services/web/server/src/simcore_service_webserver/reverse_proxy/handlers/paraview.py

```diff
@@ -10,20 +10,22 @@
 
 import aiohttp
 from aiohttp import client, web
+from yarl import URL
 
-SUPPORTED_IMAGE_NAME = "DISABLED" #"simcore/services/dynamic/3d-viewer"
+SUPPORTED_IMAGE_NAME = "simcore/services/dynamic/3d-viewer"
 SUPPORTED_IMAGE_TAG = "==1.0.5"
 
 logger = logging.getLogger(__name__)
 
 
-async def handler(req: web.Request, service_url: str, mount_point: str, proxy_path: str):
+async def handler(req: web.Request, service_url: str, mountpoint: str, proxy_path: str):
     # FIXME:
     assert req.path_qs.endswith(proxy_path)
-    assert mount_point in req.path, "Expected /x/identifier as mount point, got %s" % req.path
+    assert mountpoint in req.path, "Expected /x/identifier as mount point, got %s" % req.path
 
     #target_url = service_url + req.path_qs
-
+                    mountpoint, proxy_path, req.headers)
+    full_url = URL(service_url) / proxy_path
```

Comment:
is / a special kind of concatenation?

Reply:
This is  using the yarl library for URL. It is like pathlib but for URLs and brings a bit of flexibility with paths including / or not

Reply:
operator overloading ... cool!